scripts/test_maxcompute.sh

```diff
@@ -34,6 +34,9 @@ go install ./...
 # TODO(Yancey1989): enable all the unit test for the maxcompute
 SQLFLOW_log_level=debug go test -p 1 -v ./cmd/... -run TestEnd2EndMaxCompute
 
+export SQLFLOW_submitter=pai
+SQLFLOW_log_level=debug go test -p 1 -v ./cmd/... -run TestEnd2EndMaxCompute
```

Comment:
@shendiaomo `test_maxcompute.sh` won't run if the PR is filed from a fork. Could you please file this PR from a branch of this repo?

Reply:
Done. See https://travis-ci.com/sql-machine-learning/sqlflow/jobs/259314068